### PR TITLE
Dx 1401 Peer get tns redirect bug

### PR DESCRIPTION
**Pull request by BryanZWu**

### CODE REVIEW

File: lib/sipPeer.js [hunk 1]
```diff
@@ -53,15 +53,21 @@ SipPeer.prototype.delete = function(callback){
...
   var field = null;
-  if(arguments.length === 2){
-    url += "/" + number;
-  }
-  else{
+  var query = null;
+  if(arguments.length === 1){
+    query = {
+      page: 1,
+      size: 50000
```
Comment (jmulford-bw) [inline lib/sipPeer.js:60]: `5000` may be a better default size, I think that's what iris generally uses for TNs

File: lib/sipPeer.js [hunk 2]
```diff
@@ -53,15 +53,21 @@ SipPeer.prototype.delete = function(callback){
...
-    url += "?" + "page=1&size=50000" 
+    field = "sipPeerTelephoneNumbers";
+  } else if(typeof number === "string" || typeof number === "number" || number instanceof String){
+    url += "/" + number;
+  } else {
+    query = number; 
     field = "sipPeerTelephoneNumbers";
   }
-  this.client.makeRequest("get", url, function(err, res){
+  this.client.makeRequest("get", url, query, function(err, res){
```
Comment (jmulford-bw) [inline lib/sipPeer.js:70]: Can you add a check to make sure that `query` always has `page` and `size`, and set those if not? That way if someone uses a single different query parameter it'll still work as expected

https://github.com/Bandwidth/node-numbers/blob/master/lib/tn.js#L29 has a similar check

Comment (BryanZWu) [reply lib/sipPeer.js:72]: Gotcha, took care of it

File: lib/sipPeer.js (hunk 1 above)
Comment (BryanZWu) [reply lib/sipPeer.js]: done

**Pull request merged**